examples/pylab_examples/image_masked.py

```diff
@@ -38,24 +40,33 @@
 # Anything above that range is colored based on palette.set_over, etc.
 
 # set up the axes
-fig, (ax1, ax2) = plt.subplots(1, 2)
+fig, (ax1, ax2) = plt.subplots(2, 1, figsize=(6, 5.4))
```

Comment:
Isn't the second argument unecessary?
I also tend to favor being explicit : `nrows=2`.